1-documentation/doc-alert/doc-alert.css

```diff
@@ -0,0 +1,29 @@
+@import "../../for-import/main.css";
+
+.wrapper-doc {
+    text-align: center;
+    margin: 0;
+    padding: 0;
+}
+
+.embed-wrapper {
+    width: auto;
+    height: 200px;
+    margin: 2rem auto 1rem auto;
+    overflow-x: auto;
+    overflow-y: auto;
+}
+
+iframe {
+    overflow-x: auto;
+    overflow-y: visible;
+}
+
+
+
+@media only screen and (min-width: 600px) {
+    .embed-wrapper {
+        width: auto;
+        height: auto;
+    }
+}
```

Comment:
Add new line at the end of document

Reply:
Hi Kevin, just curious, why we need to add a new line at the end of the document?